src/github.rs

```diff
@@ -13,45 +13,176 @@ use crate::{database::Database, github::{
     pull_requests::PullRequestsRepositoryPullRequestsNodesReviewRequestsNodesRequestedReviewer::User,
 }, settings::Repository as RepoInfo};
 
+pub(crate) use pull_requests::{PullRequestState, PullRequestReviewState};
+
 const GITHUB_FETCH_SIZE: i64 = 10;
 const GITHUB_URL: &str = "https://api.github.com/graphql";
 const APP_USER_AGENT: &str = concat!(env!("CARGO_PKG_NAME"), "/", env!("CARGO_PKG_VERSION"),);
 const INIT_TIME: &str = "1992-06-05T00:00:00Z";
 
-type DateTime = String;
-
+type DateTime = ChronoDateTime<Utc>;
```

Comment:
Since we defined `type DateTime = ChronoDateTime<Utc>;`, we should consistently use the `DateTime` alias across our codebase instead of directly using `ChronoDateTime<Utc>`.

Additionally, since `DateTime` is used in `graphql::pull_request.rs`, we can consider defining it with `pub(crate)` visibility to clearly indicate internal shared usage.